plugins/package-managers/python/src/main/kotlin/Pip.kt

```diff
@@ -98,7 +98,7 @@ class Pip(
         return listOf(ProjectAnalyzerResult(project, packages))
     }
 
-    private fun runPythonInspector(definitionFile: File): PythonInspector.Result {
+    internal fun runPythonInspector(definitionFile: File): PythonInspector.Result {
```

Comment:
Commit message:
- "based on / and for" -> "based on / for" (I guess)
- "meta-data" -> "metadata"
- "sibling" -> "siblings" (I guess; also remove the double whitespace afterwards)